projects/log_listener/src/log_listener/listener.py

```diff
@@ -14,17 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Log listener module."""
-import os
-import threading
 import json
 import logging
+import os
 import pathlib
+import threading
 import time
 from typing import Optional
+
 from eiffellib.events import EiffelTestExecutionRecipeCollectionCreatedEvent
+
 from .log_subscriber import LogSubscriber
 
 
+# TODO: Temporarily using two files, one for logs and one for events.
+# the log file shall be removed when the old endpoint is being removed.
```

Comment:
NIT: Better to be explicit about what to remove. If we were to add alot more endpoints before fixinf this todo knowing which is the old one becomes a bit ambigous.
```suggestion
# the log file shall be removed when the old (/log) endpoint is being removed.
```